### ISSUE TITLE
Open with browser / SVN links

### ISSUE THREAD

**Issue by RocusHalbasch** [danmunn/redmine_mylyn_connector#10]
I really haven't taken the time to look into whether this is on your side or if it's a problem with the eclipse Mylyn extension, but either way it's a problem that a solution should be found for. When right clicking on a task in the task list and selecting "Open With Browser" the url that Mylyn attempts to open is http://redmine/issues/show/5304 which returns a 404 because redmine removed the show part of the url a while back as it was not usefull, so the correct url is now http://redmine/issues/5304. There is a similar problem when parsing and linking svn repository links. The url used for svn links is compleatly wrong and cross-project links are not even recognized. I will provide more details on this if it is on your side, however if it is a problem with the eclipse extension then someone should consider updating it.

**Comment by mbuchner** [comment]
Yup - we have the same issues on our side - shouldn't be much work to just fix the wrong URL or?

**Comment by cforce** [comment]
Same here, Yep fixing the url shall do it.

**Comment by jonnsn** [comment]
+1
